modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -270,6 +270,35 @@ var result = await authService.Logout(authService.CurrentUser());
 
 NOTE: To perform *backchannel or federated logouts*, you must enable the Backchannel Logout option for the federated identity provider. More information is available in the Keycloak documentation under link:https://www.keycloak.org/docs/3.3/server_admin/topics/identity-broker/oidc.html[OICD Identity Providers, window="_blank"].
 
+== Using auth with backend
+
+To authenticate service on existing backend server users need to supply additional token:
```

Comment:
In a typical scenario, you not only want the user to authenticate on the mobile app, you also want that user to authenticate with a back end service. To achieve this, you must add an additional token:

question: where does the user add this? where do they get the token from?